src/components/dashboard/burn_tokens.tsx

```diff
@@ -372,14 +339,18 @@ export default function BurnTokens(props: mintProps) {
 		if (isNaN(val) || val.trim() === '' || Number(val) > maxWithdraw) {
 			return
 		}
-
-		const liquidityReceived = parsedAmount / (1 - totalFeePercent) // full mint-amount
-
-		setLiquidityReceived(parseUnits(liquidityReceived.toFixed(defaultDecimals), defaultDecimals)) 
+		const liquidityReceivedBig = mulPercent(parseUnitsBnJs(parsedAmount, defaultDecimals), 1/(1-totalFeePercent))
+		setLiquidityReceived(liquidityReceivedBig) 
 
 		// calculate ibc burn amount
+		const currentInverseTokenSupplyBig = BigNumber.from(bondingCurveParams.inverseTokenSupply)
 		const currentInverseTokenSupply = Number(ethers.utils.formatUnits(bondingCurveParams.inverseTokenSupply, inverseTokenDecimals.toString()))
 		const k = 1 - curveUtilization
```

Comment:
also, contract is using invariant and utilization for calculation rather than m & k now

Reply:
yea for the mint case, we use the invariant/utilization formula like the contract

for burns, the contract does not have an "exactAmountOut" calculation, and just went with the m&k version; imo the formula complexity is about the same